### PR TITLE
reconciling with jsbeautify

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: linters/.jscsrc [hunk 1]
```diff
@@ -7,25 +7,12 @@
...
-  },
-  "disallowSpacesInFunctionDeclaration": {
-    "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInFunctionExpression": {
-    "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInNamedFunctionExpression": {
-    "beforeOpeningRoundBrace": true
-  },
```
Comment (gvn) [inline linters/.jscsrc]: These spacing rules are automatically enforced and validated by jsbeautify.

File: linters/.jscsrc [hunk 2]
```diff
@@ -7,25 +7,12 @@
...
   "disallowTrailingComma": true,
   "disallowTrailingWhitespace": true,
   "disallowYodaConditions": true,
   "maximumLineLength": 120,
   "requireBlocksOnNewline": 1,
-  "requireCamelCaseOrUpperCaseIdentifiers": true,
+  "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
```
Comment (gvn) [inline linters/.jscsrc]: `true` is really strict and makes using 3rd party APIs really difficult if they don't adhere to this convention.

**Comment by gvn** [comment]
@simonwex A few more tweaks! :zap:

**Comment by simonwex** [comment]
:shipit:

**Pull request merged**